_config/site.yml

```diff
@@ -98,6 +98,7 @@ profiles:
     js_minifier: disabled
     html_minifier: disabled
     legacy_post_syntax_highlighting: disabled
+    ignore_older_than_days: 30
```

Comment:
I don't know whether that's possible (can't find anything in the docs, so probably not). In development it's set to one year, but of course you can alter it there as you like.